exporter/datadogexporter/factory.go

```diff
@@ -51,12 +51,6 @@ var _ = featuregate.GlobalRegistry().MustRegister(
 	featuregate.WithRegisterToVersion("v0.129.0"),
 )
 
-var metricExportNativeClientFeatureGate = featuregate.GlobalRegistry().MustRegister(
```

Comment:
Can we do the following instead of removing it?

```go
var _ = featuregate.GlobalRegistry().MustRegister(
	"exporter.datadogexporter.metricexportnativeclient",
	featuregate.StageStable,
	featuregate.WithRegisterToVersion("v0.135.0"),
	featuregate.WithRegisterDescription("When enabled, metric export in datadogexporter uses native Datadog client APIs instead of Zorkian APIs."),
)
```

This will give a slightly nicer error message for users.